app/controllers/api/v1/owners_controller.rb

```diff
@@ -25,8 +25,9 @@ def create
   def destroy
     owner = @rubygem.owners.find_by_name(params[:email])
     if owner
-      if @rubygem.ownerships.find_by_user_id(owner.id).try(:destroy)
-        render plain: "Owner removed successfully."
+      ownership = @rubygem.ownerships.find_by(user_id: owner.id)
+      if ownership&.safe_destroy
```

Comment:
Although I have switched back to using `try` here, we have been using safe navigation operator for a quite a while now. I doubt codebase supports older rubies.